### ISSUE TITLE
deleting keyframes on OF 0.11.2 

### ISSUE THREAD

**Issue by stephanschulz** [roymacdonald/ofxLineaDeTiempo#12]
i am noticing that when running the `example-basic` i am not able to delete a keyframe. i click on it to select it, a small circle is drawn in it's center, i press the delete or back-space key but the keyframe does not go away.
BUT if i create a new keyframe by clicking anywhere on a timeline, i can now delete any keyframe for this particular ofParameter.

is this normal?

**Comment by roymacdonald** [comment]
ha. No. it shouldn't behave like that. I will check.

**Comment by roymacdonald** [comment]
I checked and it only happens with the first keyframe. I am allowed delete the others.
Is it still happening to you?

**Comment by stephanschulz** [comment]
here an example on how it does not work for me on OF0.11.2
https://www.dropbox.com/s/ajh9eevvtimifhu/Screen%20Recording%202023-02-08%20at%2012.42.24%20PM.mov?dl=0

**Comment by roymacdonald** [comment]
Right. I noticed that. It must be staying in a weird state when you load.
I'll fix it asap
